tests/databases/test_snowflake.py

```diff
@@ -200,8 +200,8 @@ def test_if_exist_param_of__load_pandas_dataframe_to_table(database_table_fixtur
 def test_load_file_to_table(database_table_fixture):
     """Test loading on files to snowflake database"""
     database, target_table = database_table_fixture
-    filepath = str(pathlib.Path(CWD.parent, "data/sample.csv"))
-    database.load_file_to_table(File(filepath), target_table, {})
+    filepath = str(pathlib.Path(CWD.parent, "data/sub_folder/"))
```

Comment:
We need an integration test for checking this behaviour for remote storage like S3 or GCS too

Reply:
Added it here: https://github.com/astronomer/astro-sdk/pull/740/commits/544a21cb4982af2184337c4e461c258715b8c397

I have uploaded the similar folder structure on s3 bucket